src/main/java/com/sulphate/chatcolor2/commands/ChatColorCommand.java

```diff
@@ -788,7 +700,7 @@ public String getColor(String str) {
         }
     }
 
-    public String getModifier(String str) {
+    private static String getModifier(String str) {
```

Comment:
Just wondering, why private static and not just private? I may be being stupid.

Reply:
Because by doing so I can call that method from other static methods of the class. If it wasn't static, I would need to get an instance of it beforehand, and that adds some complexity. Also, in this particular case, "getting a modifier" can be thought as an operation common to every instance of the class, which doesn't need to be associated with any object in particular, because it doesn't use any of its non-static attributes.

Reply:
Got it. Thanks :) Was just me being silly.